src/EventLoop/Internal/DriverSuspension.php

```diff
@@ -19,7 +19,7 @@ final class DriverSuspension implements Suspension
     /** @var \WeakReference<\Fiber>|null */
     private readonly ?\WeakReference $fiberRef;
 
-    private ?\FiberError $fiberError = null;
+    private ?\Throwable $exception = null;
```

Comment:
Since PHP makes a distinction between `Throwable` (PHP-land) and `Exception` (user-land) errors, and since in this case we're accepting the base type `Throwable`, wouldn't it be better to name this `$throwable` accordingly? Applies throughout this file.

Reply:
I tend to use `$exception` and `$e` in catch blocks, mostly for historical reasons, though both `Exception` and `Error` are typically referred to collectively as exceptions. Still, `$throwable` does make more sense when catching/holding either.